playground/nextjs/src/pages/render-notification/index.tsx

```diff
@@ -8,7 +8,7 @@ export default function Home() {
       <Title title="Render Notification Props" />
       <Inbox
         options={novuConfig}
-        renderNotification={(notification) => {
+        renderNotification={({ notification }) => {
```

Comment:
changed it to this interface as it's easier to work / extend in the future it would be needed